adhocteam/Head-Start-TTADP
# Add ability to delete a report in My Alerts

**Description of change**
In the My Alerts section, theres a new column for the context menu which allows the user to view the report or delete it.

**How to test**
Pull down changes and run locally.
Go to My Alerts and you should see the new context menu.
Clicking Delete should soft delete in the backend and remove the report from the view

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-83

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
